lib/session/call-session.js

```diff
@@ -640,14 +659,16 @@ class CallSession extends Emitter {
    * Check for speech credentials for the specified vendor
    * @param {*} vendor - google or aws
    */
-  getSpeechCredentials(vendor, type) {
+  getSpeechCredentials(vendor, type, label = null) {
     const {writeAlerts, AlertType} = this.srf.locals;
     if (this.accountInfo.speech && this.accountInfo.speech.length > 0) {
-      const credential = this.accountInfo.speech.find((s) => s.vendor === vendor);
+      const credential = this.accountInfo.speech.find((s) => s.vendor === vendor &&
+        ((label && s.label === label) || label === null));
       if (credential && (
         (type === 'tts' && credential.use_for_tts) ||
         (type === 'stt' && credential.use_for_stt)
       )) {
+        this.logger.info(`Speech credential vendor: ${credential.vendor} and Label: ${credential.label} is chosen`);
```

Comment:
I'd prefer the info message not to mention label if label is null

Reply:
fixed